### DISCUSSION TITLE
升级最新版本以后流量上行变多了

### DISCUSSION THREAD

**Discussion by 99xx9** [midoks/mdserver-web#575] [General]
升级最新版本以后流量上行变多了
但是查看真实流量并没有变多，是不是计算显示变错了

**Comment by midoks** [comment]
<img width="597" alt="截屏2024-05-14 14 31 42" src="https://github.com/midoks/mdserver-web/assets/4169529/65d329bd-536e-4bba-ada6-5b63bb15f86b">

是所有流量统计，你可以选择显示lo是本地 ，enX是网络端口。
